components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java

```diff
@@ -342,6 +354,17 @@ public static boolean useOrganizationRolesForValidation(String organizationId) {
         return true;
     }
 
+    /**
+     * Return whether the given organization is a sub organization.
```

Comment:
```suggestion
     * Return whether the given organization is a sub-organization.
```
Change other places aswell

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/65/commits/13e966198010e05a2e60d6a7cd7cc301d9ed0c9a#diff-96228f2c7b6a184e56f8c4f5b0ec5878c01dc60313cbf7ac79ee197005edf5b6